### DISCUSSION TITLE
Ambient Sound

### DISCUSSION THREAD

**Discussion by JAyres88** [Xeno69/Domination#467] [Ideas]
Hey guys, is there any way of turning off the gunfire theme that plays when we enter the AO? Is it possible to add as mission param?

**Comment by Xeno69** [comment]
Always check the existing Paramus

https://github.com/Xeno69/Domination/wiki/Params#d_withambientradio--with-ambient-radio

**Comment by longtimegamer** [comment]
Xeno pointed out the setting for radios.

For the gunfire sound you are describing, I added a new setting that is not yet released.  It is already in the code and will be in the next release.

d_noambient_bf_sounds_setting = 1

**Reply by JAyres88** [reply]
Nice, thanks bro!
